src/components/App.tsx

```diff
@@ -34,600 +35,433 @@ import Account from "./Account/Account";
 
 interface IProps {
+  // eslint-disable-next-line no-unused-vars
   addTestResults: (result: any) => void;
+  // eslint-disable-next-line no-unused-vars
   addFinalResultCallback: (result: any) => void;
   clearFinalResult: () => void;
+  // eslint-disable-next-line no-unused-vars
   setDeployedResult: (result: any) => void;
-  clearDeployedResult: () => void;
   setCallResult: (result: any) => void;
   setAccountBalance: (accData: any) => void;
   setCurrAccChange: (accData: any) => void;
   setTestNetId: (testNetId: any) => void;
-  test: any;
   accountBalance: number;
   accounts: string[];
   currAccount: IAccount;
   testNetId: string;
+  test: any;
 }
 
-interface IState {
-  message: any[];
-  compiled: any;
-  error: Error | null;
-  fileName: any;
-  contractName: any;
-  processMessage: string;
-  availableVersions: any;
-  gasEstimate: number;
-  deployedResult: string;
-  tabIndex: number;
-  txTrace: object;
-  traceError: string;
-  accounts: string[];
-  currAccount: IAccount | undefined;
-  balance: number;
-  transactionResult: string;
-  testNetId: string;
-  fileType: string;
-  selctorAccounts: any;
-  contracts: any;
-  files: any;
-  accountName: IAccount;
-  testNets: object[];
-  localAcc: any[];
-  testNetAcc: any[];
-  appRegistered: boolean;
-}
 interface IOpt {
   value: string;
   label: string;
 }
 // @ts-ignore
 const vscode = acquireVsCodeApi(); // eslint-disable-line
-class App extends Component<IProps, IState> {
-  public state: IState;
+const App = (props: IProps) => {
+  const [message, setMessage] = useState<any[]>([]);
+  const [compiled, setCompiled] = useState<any>({});
+  const [error, setError] = useState<Error | null>(null);
+  const [fileName, setFileName] = useState<any>("");
+  const [contractName, setContractName] = useState<any>("");
+  const [processMessage, setProcessMessage] = useState("");
+  const [availableVersions, setAvailableVersions] = useState<any[]>([]);
+  const [gasEstimate, setGasEstimate] = useState(0);
+  const [deployedResult, setDeployedResult] = useState("");
+  const [tabIndex, setTabIndex] = useState(0);
+  const [txTrace, setTxTrace] = useState({});
+  const [accounts, setAccounts] = useState<any[]>([]);
+  const [selctorAccounts, setSelctorAccounts] = useState<IAccount[]>([]);
+  const [contracts, setContracts] = useState<any[]>([]);
+  const [files, setFiles] = useState<any[]>([]);
+  const [currAccount, setCurrAccount] = useState<IAccount>();
+  const [balance, setBalance] = useState(0);
+  const [transactionResult, setTransactionResult] = useState("");
+  const [testNetId, setTestNetId] = useState("");
+  const [fileType, setFileType] = useState("");
+  const [traceError, setTraceError] = useState("");
+  const [accountName, setAccountName] = useState<IAccount>({
+    label: "",
+    value: [""],
+  });
+  const [localAcc, setLocalAcc] = useState<any[]>([]);
+  const [testNetAcc, setTestNetAcc] = useState<any[]>([]);
+  const [testNets, setTestNets] = useState([
```

Comment:
We probably wont need `setTestNets`. Use `[testNets, setTestNets] = useState(...)` instead.